packages/react/src/components/Popup/Popup.tsx

```diff
@@ -132,6 +133,7 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
 
   static slotClassNames: PopupSlotClassNames = {
     content: `${Popup.className}__content`,
+    trigger: `${Popup.className}__trigger`,
```

Comment:
I am not sure that we really need this because we don't apply any styles for `trigger`. `trigger` is not also a slot.